spec/sitemap_generator/link_set_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe SitemapGenerator::LinkSet do
-  let(:default_host) { 'http://example.com' }
+  let(:default_host) { 'http://example.com/' }
```

Comment:
Please revert all the changes to this file.  The hostname must be able to be entered with or without a trailing slash, so having it omit the slash tests the code.  And I'd rather not use `URI.join` in case that were to mask an issue with the joining of the path fragments.